components/library/glibmm/glibmm.p5m

```diff
@@ -27,1745 +28,499 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/class_opaque_copyable.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/class_opaque_refcounted.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/class_shared.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/compare.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/convert.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/convert_base.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/convert_gio.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/convert_glib.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/convert_glibmm.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/ctor.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/doc.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/enum.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/gerror.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/initialize.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/initialize_base.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/initialize_gio.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/initialize_glib.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/initialize_glibmm.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/member.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/method.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/property.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/signal.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/m4/vfunc.m4
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/DocsParser.pm
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/Enum.pm
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/Function.pm
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/FunctionBase.pm
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/GtkDefs.pm
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/Object.pm
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/Output.pm
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/Property.pm
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/Util.pm
+file path=usr/lib/$(MACH64)/glibmm-2.68/proc/pm/WrapParser.pm
+link path=usr/lib/$(MACH64)/libgiomm-2.68.so target=libgiomm-2.68.so.1
+link path=usr/lib/$(MACH64)/libgiomm-2.68.so.1 target=libgiomm-2.68.so.1.3.0
+file path=usr/lib/$(MACH64)/libgiomm-2.68.so.1.3.0
+link path=usr/lib/$(MACH64)/libglibmm-2.68.so target=libglibmm-2.68.so.1
+link path=usr/lib/$(MACH64)/libglibmm-2.68.so.1 target=libglibmm-2.68.so.1.3.0
+file path=usr/lib/$(MACH64)/libglibmm-2.68.so.1.3.0
+link path=usr/lib/$(MACH64)/libglibmm_generate_extra_defs-2.68.so \
+    target=libglibmm_generate_extra_defs-2.68.so.1
+link path=usr/lib/$(MACH64)/libglibmm_generate_extra_defs-2.68.so.1 \
+    target=libglibmm_generate_extra_defs-2.68.so.1.3.0
+file path=usr/lib/$(MACH64)/libglibmm_generate_extra_defs-2.68.so.1.3.0
+file path=usr/lib/$(MACH64)/pkgconfig/giomm-2.68.pc
+file path=usr/lib/$(MACH64)/pkgconfig/glibmm-2.68.pc
+file path=usr/lib/giomm-2.68/include/giommconfig.h
```

Comment:
Because of the major so-name changes we'll need to rebuild or upgrade the dependent packages